src/platform/STM32/timer_stdperiph.c

```diff
@@ -838,7 +838,7 @@ void timerInit(void)
     memset(timerConfig, 0, sizeof(timerConfig));
 
 #if defined(PARTIAL_REMAP_TIM3)
-    GPIO_PinRemapConfig(GPIO_PartialRemap_TIM3, ENABLE);
+    GPIO_PinRemautopilotConfig(GPIO_PartialRemap_TIM3, ENABLE);
```

Comment:
```suggestion
    GPIO_PinRemapConfig(GPIO_PartialRemap_TIM3, ENABLE);
```

Reply:
Thanks, my bad.  Very grateful you found this.